src/sqlfluff/core/templaters/jinja.py

```diff
@@ -303,8 +303,8 @@ def process(
             )
             return None, violations
 
-    re_open_tag = re.compile(r"^{%[\+\-]?\s*")
-    re_close_tag = re.compile(r"\s*[\+\-]?%}$")
+    re_open_tag = re.compile(r"^\s*{%[\+\-]?\s*")
```

Comment:
Unrelated bug fix turned up by a dbt test: Sometimes the Jinja tags are preceded or followed by whitespace.